charts/MapTab.tsx

```diff
@@ -288,6 +294,7 @@ export class MapTab extends React.Component<MapTabProps> {
                         defaultFill={map.data.colorScale.noDataColor}
                         mapToDataEntities={map.data.mapToDataEntities}
                         formatYear={map.data.formatYear}
+                        headerHeight={layout.header.height}
```

Comment:
The map ideally shouldn't need to know how big the header above it is. Are we passing it because the tooltip location is relative to the chart bounding box instead of the boundary of the map?

We started using `<ControlOverlay>` to get around this problem – the overlay is always relative to the plot, not the chart box.